src/MsTestTrxLogger/MsTestTrxLogger.cs

```diff
@@ -1,21 +1,36 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.IO;
 using Microsoft.VisualStudio.TestPlatform.ObjectModel;
 using Microsoft.VisualStudio.TestPlatform.ObjectModel.Client;
 
 namespace MsTestTrxLogger
 {
     [ExtensionUri("logger://MsTestTrxLogger/v1")]
     [FriendlyName("MsTestTrxLogger")]
-    public class MsTestTrxLogger : ITestLogger
+    public class MsTestTrxLogger : ITestLoggerWithParameters
     {
+        /// <summary>
+        /// Cache the TRX file path
+        /// </summary>
+        private string trxFilePath;
+
+        /// <summary>
+        /// Parameters dictionary for logger. Ex: {"LogFileName":"TestResults.trx"}.
+        /// </summary>
+        private Dictionary<string, string> parametersDictionary;
+        private string LogFileNameKey = "LogFileName";
+        private string testResultsDirPath;
+
         public void Initialize(TestLoggerEvents events, string testRunDirectory)
```

Comment:
This `Initialize` method is not called by the framework any more, but only we call it from the new `Initialize`, right?  
In this case, please make this private, and move it under the public one.